.github/workflow/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: ci
```

Comment:
This file should only be used for testing purposes, we should use the build-docs.yml for documentation configurations.

Reply:
This is how its set up from the mkdocs site, stayed with their structure. Will have a look if it causes any issues if I change it.

Reply:
Done